dnf/cli/commands/repoquery.py

```diff
@@ -363,6 +363,15 @@ def run(self):
         q = self.filter_repo_arch(self.opts, q)
         orquery = q
 
+        if self.opts.obsoletes:
+            out = set()
+            for pkg in q:
+                for i in pkg.obsoletes:
+                    out.add(str(i))
```

Comment:
OK, I haven't taken that combination into account. I will make new push tomorrow.